scripts/r.reclass.area/r.reclass.area.py

```diff
@@ -163,18 +168,29 @@ def reclass(inf, outf, lim, clump, diag, les):
     p1.wait()
     p2.stdin.close()
     p2.wait()
+    expression = "$outfile = $recfile"
+    mapcalc_kwargs = {"outfile": outfile, "recfile": recfile}
     if p2.returncode != 0:
         if lesser:
-            grass.fatal(
-                _("No areas of size less than or equal to %f " "hectares found.")
-                % limit
+            op_str = "less"
+        else:
+            op_str = "greater"
+        if nodata is True:
+            grass.warning(
```

Comment:
You are probably in the best position to decide that, but if user requests NULL raster in those cases, is warning needed? Isn't that a verbose message instead? In other words, the new flag seems to make the the "no areas" state to be a standard state with appropriate output, so why a warning?